submissions/krylenger/html-css-popup/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pop-up</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+    <nav class="navigation-container">
+        <div class="navigation-list-container">
+            <ul class="navigation-list">
+                <li><a href="https://www.gmail.com/">Mail</a></li>
```

Comment:
Correctly to use the class name of elements as like as [BEM](https://ru.bem.info/methodology/css/#%D0%BA%D0%B0%D0%BA-%D0%BF%D0%B5%D1%80%D0%B5%D0%B9%D1%82%D0%B8-%D0%BD%D0%B0-css-%D0%BF%D0%BE-%D0%B1%D1%8D%D0%BC) methodology

Reply:
Thank you for introducing this technology to me. I've dedicated some time today to learn about this methodology and I definitely like it. 
My main concern now - to implement the key principles of this methodology (such as encapsulation, DRY, mix, etc)  - will require to rewrite almost all my css file because now I have pretty nested and dependent structure with many elements depend on others. 
Can I just implement renaming of my classes according to methodology and start to use it mainly in my next project. 
Because as far as I see it may be a pretty complicated task and I am afraid to get lost without any practice background of this methodology.

Reply:
It is only my recommendation.
Difficult to read and understand what certain elements mean.

Reply:
What's the best practice naming elements inside long and nested structure? To split main block into several nested blocks (even if we are not going to use these blocks in different places) and then create elements from these new blocks? (even if we lose the track of relation these items to main block)

Reply:
Did I get it right?

Reply:
@A-Ostrovnyy, could you add corrections to our discussion?

Reply:
Use BEM or not is up to you.
If you have a lot of nested blocks, you apparently need to separate them logically: if you can reuse this block in a different place, create a new block, don't create a new block only for the new block
[Read more about BEM](https://ru.bem.info/methodology/quick-start/) if you have a question don't be shy to ask here or telegram)

Reply:
thanks to everyone

Reply:
✅ renaming classes using BEM methodology